app/dependencies/rnn.py

```diff
@@ -22,6 +21,8 @@ def train_lstm(currency_name):
 
     if historical_json_data:
         historical_data = get_each_currency_data(historical_json_data, currency_name)
+    else:
```

Comment:
no need of else block

Reply:
Agar else na lagao to historical_json_data ke empty hone par bhi code aage chala jata we had this error in last logs.